Jenkinsfile

```diff
@@ -83,6 +83,8 @@ pipeline {
 			}
                     }
                     steps {
+			echo 'Build all the examples in the book'
```

Comment:
we should make this its own stage probably

Reply:
it needs the built libraries, but I don't wanna compile twice. Perhaps we make a shared artifact (or the Jenkins term for it) and copy the target directory contents across?